modules/ROOT/pages/customsidebar.adoc

```diff
@@ -78,7 +78,10 @@ tinymce.init({
       icon: 'comment',
       onSetup: (api) => {
         console.log('Render panel', api.element());
-      },
+        return () => {
+          console.log('Removing sidebar');
+        };
+      };
```

Comment:
```suggestion
      },
```
Oops somehow this has been converted into a semicolon when it should be a comma.

Reply:
That could be a typo on my part. Commiting.